cmd/lotus-miner/sectors.go

```diff
@@ -813,6 +813,10 @@ var sectorsRenewCmd = &cli.Command{
 			Usage: "use up to this amount of FIL for one message. pass this flag to avoid message congestion.",
 			Value: "0",
 		},
+		&cli.Int64Flag{
+			Name:  "max-sectors",
+			Usage: "The maximum number of sectors contained in each message message",
```

Comment:
Nit: Other usage strings here start lower-cased
```suggestion
			Usage: "the maximum number of sectors contained in each message message",
```